sdk/monitor/azure-monitor-query/samples/champion_scenarios.md

```diff
@@ -0,0 +1,288 @@
+## Azure Monitor Query Champion Scenarios
+
+This document covers the basic champion Scenarios to use the package.
+
+### Authenticate the client
+
+Consider the following example, which creates and authenticates clients for both logs and metrics querying:
+
+```python
+from azure.identity import DefaultAzureCredential
+from azure.monitor.query import LogsQueryClient, MetricsQueryClient
+
+credential = DefaultAzureCredential()
+logs_client = LogsQueryClient(credential)
+metrics_client = MetricsQueryClient(credential)
+```
+
+### Make a simple query to the service
+
+* Each row is converted into a native python data type. For example, time is a datetime object instead of string.
+
+#### Results in tabular form
+
+```python
+import os
+import pandas as pd
+from datetime import timedelta
+from azure.monitor.query import LogsQueryClient
+from azure.identity import DefaultAzureCredential
+
+def query():
+    query = """AppRequests |
+    summarize avgRequestDuration=avg(DurationMs) by bin(TimeGenerated, 10m), _ResourceId"""
+
+    response = client.query(os.environ['LOG_WORKSPACE_ID'], query, 
+    timespan=timedelta(days=1))
+
+    if not response.tables:
+        return None
+
+    primary_table = response.tables[0]
+    df = pd.DataFrame(table.rows, columns=table.columns)
+    return df
+
+if __name__ == '__main__':
+    print(query())
+
+"""
+    TimeGenerated                                        _ResourceId          avgRequestDuration
```

Comment:
nit:
I'm assuming that this was copied from the actual resulting output, but for some reason, the indenting here is different from, for example, line 165 and 177. Do you know why this is?